commit 01a4684c73aaab687ac8626e640b89906529c101
Author: Michal Kubecek <mkubecek@suse.cz>
Date:   Sat Aug 8 22:53:50 2020 +0200

    vmmon: handle change of vmstat slab counters to bytes
    
    Mainline commit d42f3245c7e2 ("mm: memcg: convert vmstat slab counters to
    bytes") in kernel 5.9-rc1 changed vmstat accounting of slab to byte
    counters. Unfortunately there doesn't seem to be any obvious way to check
    for this change except version check.

Index: work/vmmon-only/linux/hostif.c
===================================================================
--- work.orig/vmmon-only/linux/hostif.c
+++ work/vmmon-only/linux/hostif.c
@@ -1606,7 +1606,9 @@ HostIF_EstimateLockedPageLimit(const VMD
 #else
    unsigned int lockedPages = global_page_state(NR_PAGETABLE) +
 #endif
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 13, 0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 9, 0)
+                              global_node_page_state_pages(NR_SLAB_UNRECLAIMABLE_B) +
+#elif LINUX_VERSION_CODE >= KERNEL_VERSION(4, 13, 0)
                               global_node_page_state(NR_SLAB_UNRECLAIMABLE) +
 #else
                               global_page_state(NR_SLAB_UNRECLAIMABLE) +
